app/helpers/movies_helper.rb

```diff
@@ -67,9 +67,7 @@ def full_cast_link(movie)
   end
 
   def runtime_display(movie)
-    if movie.runtime.present?
-      movie.runtime % 60 == 0 ? "#{movie.runtime/60}hr" : "#{movie.runtime/60}hr #{movie.runtime % 60}min"
-    end
+    DateAndTimeHelper.display_time(movie.runtime) if movie.runtime.present?
   end
 
```

Comment:
The new `DateAndTimeHelper` handles all calculations about dates and times. That is why all of this logic was moved from the `movies_helper`. This new file refactor happened when we needed to calculate a person's age.